libr/anal/data.c

```diff
@@ -159,10 +159,8 @@ R_API RAnalData *r_anal_data_new_string (ut64 addr, const char *p, int len, int
 	ad->type = type;
 	if (len == 0)
 		len = strlen (p);
-	if (type == R_ANAL_DATA_TYPE_WIDE_STRING) {
-		/* TODO: add support for wide strings */
-		//eprintf ("r_anal_data_new_string: wide string not supported yet\n");
-	} else {
+	/* TODO: add support for wide strings */
+	if (type != R_ANAL_DATA_TYPE_WIDE_STRING) {
```

Comment:
can you leave an elseif with a todo in it instead of just the comment? makes it easier to know "code goes here"